2019-03-27 12:15:46 -0 - [main] ERROR com.cpgemini.bankapp.client.BankAppClient  -Exception :
 com.capgemini.bankapp.exception.LowBalanceException: You dont have sufficient fund
	at com.capgemini.bankapp.service.impl.BankAccountServiceImpl.withdraw(BankAccountServiceImpl.java:36)
	at com.cpgemini.bankapp.client.BankAppClient.main(BankAppClient.java:70)
2019-03-27 12:20:23 -0 - [main] ERROR com.cpgemini.bankapp.client.BankAppClient  -Exception : 
 com.capgemini.bankapp.exception.LowBalanceException: You dont have sufficient fund
	at com.capgemini.bankapp.service.impl.BankAccountServiceImpl.withdraw(BankAccountServiceImpl.java:36)
	at com.capgemini.bankapp.service.impl.BankAccountServiceImpl.fundTransfer(BankAccountServiceImpl.java:70)
	at com.cpgemini.bankapp.client.BankAppClient.main(BankAppClient.java:100)
2019-03-28 11:47:06 -0 - [main] ERROR com.capgemini.bankapp.util.DbUtil  -SQLException 
 com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Connection.close() has already been called. Invalid operation in this state.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.Util.getInstance(Util.java:381)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:985)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:927)
	at com.mysql.jdbc.ConnectionImpl.getMutex(ConnectionImpl.java:3015)
	at com.mysql.jdbc.ConnectionImpl.commit(ConnectionImpl.java:1521)
	at com.capgemini.bankapp.util.DbUtil.commit(DbUtil.java:53)
	at com.capgemini.bankapp.service.impl.BankAccountServiceImpl.addNewBankAccount(BankAccountServiceImpl.java:70)
	at com.cpgemini.bankapp.client.BankAppClient.main(BankAppClient.java:57)
